src/admin-views/edit_admin_panel.php

```diff
@@ -0,0 +1,159 @@
+<div id="tribe_panel_edit" class="ticket_panel panel_edit" aria-hidden="true" >
+	<?php
+	/**
+	 * Allows for the insertion of additional elements into the main ticket edit panel
+	 *
+	 * @param Post ID
+	 * @since TBD
+	 */
+	do_action( 'tribe_events_tickets_pre_edit', $post_id );
+	?>
+	<?php if ( get_post_meta( $post_id, '_EventOrigin', true ) === 'community-events' ) {
+		?>
+		<?php // @TODO: this should get moved to Community Events? Use tribe_events_tickets_pre_edit ?>
```

Comment:
It _does_ seem like Community Events specific messaging should be moved to that plugin; `tribe_events_tickts_pre_edit` provides a means of injecting it. Any reason we don't just do that?